.configure

```diff
@@ -20,7 +20,7 @@
     },
     {
       "file": "android/debug.keystore",
-      "destination": "~/.android/debug_a8c.keystore",
+      "destination": "~/.configure/wordpress-android/secrets/debug_a8c.keystore",
```

Comment:
This was technically already outside the repository, but I moved it to the new default location: `~/.configure/<project-name>/secrets/`

Reply:
Thanks I appreciate the standardization 👍

I'm not familiar with how/where this keystore file is used in the codebase though, is it expected that there was nothing changed in the rest of the repo (like gradle file or properties file or project settings) to update to that new path?

Reply:
Nice! But as @AliSoftware suggested, you have to update the configuration in `gradle.properties` to match this new path in order to make the debug certificate work on local builds.